resources/qml/dialogs/RoomMembers.qml

```diff
@@ -63,6 +63,35 @@ ApplicationWindow {
             onClicked: TimelineManager.openInviteUsers(members.roomId)
         }
 
+        RowLayout {
+            spacing: Nheko.paddingMedium
+
+            MatrixTextField {
+                id: searchBar
+
+                Layout.fillWidth: true
+                placeholderText: qsTr("Search...")
+                onTextChanged: members.setFilterString(text)
+
+                Component.onCompleted: forceActiveFocus()
+            }
+
+            Label {
+                text: qsTr("Sort by: ")
+            }
+
+            ComboBox {
```

Comment:
Maybe stack those over each other. It looks a bit weird otherwise, because the search field is so narrow.